.env.sample.mainnet

```diff
@@ -2,6 +2,7 @@
 # in docker-compose.yml. Rename this file to `.env` and then uncomment and set any variable below.
 
 # Overrides network for all the relevant services.
+ETH2_NETWORK=mainnet
```

Comment:
Instead of introducing `ETH2_NETWORK`, can we re-use the `NETWORK` variable?

Reply:
Done